### ISSUE TITLE
OSError: 'grid' not found in the style library and input is not a valid URL or path;

### ISSUE THREAD

**Issue by zzzzk12345** [garrettj403/SciencePlots#19]
Is there a new version or something? why grid not available
```
with plt.style.context(['science', 'grid']):
    fig, ax = plt.subplots()
    for p in [10, 15, 20, 30, 50, 100]:
        ax.plot(x, model(x, p), label=p)
    ax.legend(title='Order')
    ax.set(xlabel='Voltage (mV)')
    ax.set(ylabel='Current ($\mu$A)')
    ax.autoscale(tight=True)
```

**Comment by xuehuachunsheng** [comment]
Please use the latest commit with `pip install git+https://github.com/garrettj403/SciencePlots.git`. 
Or installing it by `python3 setup.py install`.
It would work.

**Comment by garrettj403** [comment]
``grid`` is included in the latest release of SciencePlots (v1.0.4).

You should be able to install it [via PyPI](https://pypi.org/project/SciencePlots/): ``pip install SciencePlots==1.0.4``
